app/Http/Middleware/RedirectToStatusUpdate.php

```diff
@@ -34,6 +34,14 @@ public function handle(Request $request, Closure $next)
         if (!$user->needsUpdateTenantUntil()) {
             return $next($request);
         }
+        // Ignore those having an application in progress
+        if ($user->application) {
```

Comment:
I don't like that the existence of a php value is used to differentiate user types. First, I don't trust PHP, secondly anyone can just create an application as it is created automatically if you visit the application page. It might not make a difference here but we should aim for a clearer approach.